android/java/res/layout/activity_asset_detail.xml

```diff
@@ -279,16 +279,17 @@
                 android:text="@string/swap"
                 android:layout_marginEnd="8dp"
                 style="@style/BraveWalletButton" />
-
-            <androidx.appcompat.widget.AppCompatButton
-                android:id="@+id/btn_aurora_bridge"
-                android:layout_width="wrap_content"
-                android:layout_height="wrap_content"
-                android:text="@string/brave_wallet_aurora_button_text"
-                style="@style/BraveWalletButton"
-                android:visibility="gone"/>
         </LinearLayout>
 
+        <androidx.appcompat.widget.AppCompatButton
+            android:id="@+id/btn_aurora_bridge"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
```

Comment:
`layout_width` and `layout_height` can be removed as they are inherited by `BraveWalletButton` style .

Reply:
Yeah, though just a small unharmful nit, too small for another push and CI build, will address in another PR.